### PR TITLE
Update golang version to match what we have in the oc repo

### PR DESCRIPTION
**Pull request by soltysh**
Follow up to https://github.com/openshift/release/pull/9756
/assign @tnozicka

### CODE REVIEW

File: ci-operator/config/openshift/oc/openshift-oc-master.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```
Comment (tnozicka) [inline ci-operator/config/openshift/oc/openshift-oc-master.yaml:30]: oc has wrong images `FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder` https://github.com/openshift/oc/blob/release-4.4/images/cli/Dockerfile.rhel#L1 you need to fix that first

Comment (tnozicka) [reply ci-operator/config/openshift/oc/openshift-oc-master.yaml:30]: or just fix it here but the consistency with other configs will suffer a bit

Comment (soltysh) [reply ci-operator/config/openshift/oc/openshift-oc-master.yaml:30]: :facepalm:  yeah and that's across versions, opened https://github.com/openshift/oc/pull/494

**Comment by soltysh** [comment]
Moved to #10899 
/close

**Pull request closed without merging**